options/docker.go

```diff
@@ -70,7 +70,7 @@ func (opts *Docker) Resolve() (*client.Client, error) {
 	if opts.Host != "" && opts.Port > 0 {
 		addr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("%s:%d", opts.Host, opts.Port))
 		if err != nil {
-			return nil, errors.Wrapf(err, "could not resolve Docker daemon address %s:%d", opts.Host, opts.Port)
+			return nil, errors.Wrapf(err, "resolving Docker daemon address %s:%d", opts.Host, opts.Port)
```

Comment:
`'%s:%d'`

Reply:
I don't think we need it in this case since it wouldn't format potentially empty string.